src/test/unit/resources/eks_utils.py

```diff
@@ -0,0 +1,50 @@
+from opentelemetry.semconv.resource import (
+    ResourceAttributes,
+    CloudProviderValues,
+    CloudPlatformValues,
+)
+
+MockEksResourceAttributes = {
+    ResourceAttributes.CLOUD_PROVIDER: CloudProviderValues.AWS.value,
+    ResourceAttributes.CLOUD_PLATFORM: CloudPlatformValues.AWS_EKS.value,
+    ResourceAttributes.K8S_CLUSTER_NAME: "mock-cluster-name",
+    ResourceAttributes.CONTAINER_ID: "a4d00c9dd675d67f866c786181419e1b44832d4696780152e61afd44a3e02856",
+}
+
+
+get_cluster_info = f"""{{
```

Comment:
Usually when importing a const it's upper case and a function is a lower case
```suggestion
GET_CLUSTER_INFO = f"""{{
```